Tasks/AppCenterDistributeV2/appcenterdistribute.ts

```diff
@@ -0,0 +1,497 @@
+            if (filePath) {
+                symbolsPaths.push(filePath);
+            }
+        }
+    }
+
+    return symbolsPaths;
+}
+
+async function run() {
+    try {
+        tl.setResourcePath(path.join(__dirname, 'task.json'));
+
+        // Get build inputs
+        let apiEndpointData = getEndpointDetails('serverEndpoint');
+        let apiToken: string = apiEndpointData.authToken;
+        let apiServer: string = apiEndpointData.apiServer;
+        let apiVersion: string = apiEndpointData.apiVersion;
+
+        let userAgent = tl.getVariable('MSDEPLOY_HTTP_USER_AGENT');
+        if (!userAgent) {
+            userAgent = 'VSTS';
+        }
+        userAgent = userAgent + ' (Task:VSMobileCenterUpload)';
+
+        var effectiveApiServer = process.env['SONOMA_API_SERVER'] || apiServer;
+        var effectiveApiVersion = process.env['SONOMA_API_VERSION'] || apiVersion;
+
+        tl.debug(`Effective API Url: ${effectiveApiServer}/${effectiveApiVersion}`);
+
+        let appSlug: string = tl.getInput('appSlug', true);
+        let appFilePattern: string = tl.getInput('app', true);
+
+        /* The task has support for different symbol types but App Center server only support Apple currently, add back these types in the task.json when support is available in App Center.
+        "AndroidJava": "Android (Java)",
+        "AndroidNative": "Android (native C/C++)",
+        "Windows": "Windows 8.1",
+        "UWP": "Universal Windows Platform (UWP)"
+        */
+        let symbolsType: string = tl.getInput('symbolsType', false);
+        let symbolVariableName = null;
+        switch (symbolsType) {
+            case "Apple":
+                symbolVariableName = "dsymPath";
+                break;
+            case "AndroidJava":
+                symbolVariableName = "mappingTxtPath";
+                break;
+            case "UWP":
+                symbolVariableName = "pdbPath";
+                break;
+            default:
+                symbolVariableName = "symbolsPath";
+        }
+        let symbolsPathPattern: string = tl.getInput(symbolVariableName, false);
+        let packParentFolder: boolean = tl.getBoolInput('packParentFolder', false);
+
+        let releaseNotesSelection = tl.getInput('releaseNotesSelection', true);
+        let releaseNotes: string = null;
+        if (releaseNotesSelection === 'file') {
+            let releaseNotesFile = tl.getPathInput('releaseNotesFile', true, true);
+            releaseNotes = fs.readFileSync(releaseNotesFile).toString('utf8');
+        } else {
+            releaseNotes = tl.getInput('releaseNotesInput', true);
+        }
+
+        let isMandatory: boolean = tl.getBoolInput('isMandatory', false);
+
+        let destinations = tl.getInput('destinationIds', false) || '00000000-0000-0000-0000-000000000000';
+        tl.debug(`Effective destinationIds: ${destinations}`);
+        let destinationIds = destinations.split(/[, ;]+/); ;;;
```

Comment:
That's a lot of semicolons
```suggestion
        let destinationIds = destinations.split(/[, ;]+/);
```